### PR TITLE
adds more simple-select tests

### PR DESCRIPTION
**Pull request by tsargent**
This just adds more test coverage to simple select.

Also bumps sinon to 1.17.2, only because that was the version npm installed on this branch while @loudhearted added it as 1.17.1 on another branch at the same time:
https://github.com/tsargent/styleguide/commit/71db2a3af66cdd9b9da7e5ee88201dcb55245a39

And a very tiny change here: 
https://github.com/tsargent/styleguide/commit/f1f9d75d2b6130e0315150977d20537d423baebb

### CODE REVIEW

**Comment by johnnymo87** [comment]
Was gonna complain about missing semicolons, but I see this project doesn't have a linter.

Tests look very thorough! +1

**Comment by HipsterBrown** [comment]
Fantastic work. Love the test coverage. :+1:

File: test/forms/fields/select/simple-select.spec.js [hunk 1]
```diff
@@ -2,9 +2,179 @@ import { expect } from 'chai';
...
+
+  describe('#componentDidUpdate()', () => {
+    let callback, simple_select, options;
+    beforeEach(() => {
+      callback = Sinon.spy();
+      simple_select = TestUtils.renderIntoDocument(<SimpleSelect onChange={callback} value={'one'} options={['one','two']}/>)
+      simple_select.setState({show_options: true})
+      options = TestUtils.scryRenderedDOMComponentsWithClass(simple_select, 'simple-select-option');
+    })
+    it('fires the onChange prop when the value changes', () => {
```
Comment (jasonfellows) [inline test/forms/fields/select/simple-select.spec.js]: This is useful for all fields. Let include this for all of them as we go!

**Comment by jasonfellows** [comment]
+1, especially for tests around successful firing of prop callbacks.

There are some pending changes to border color and fieldset handling in [another pr](https://github.com/namely/styleguide/pull/233), so there will be some adjustments necessary if that passes review.

Testing for behavioral requirements are much more useful than visual ones while the project is so prone to changes.

**Comment by tsargent** [comment]
thanks for the feedback @HipsterBrown @jasonfellows @johnnymo87.  I'll merge this and watch out for any breaking changes introduced by https://github.com/namely/styleguide/pull/233

**Pull request merged**